src/mem_store/raw_col.rs

```diff
@@ -69,6 +70,10 @@ impl MixedCol {
             Arc::new(Column::null(name, self.data.len()))
         }
     }
+
+    pub fn heap_size_of_children(&self) -> usize {
+        mem::size_of::<ColType>() + self.data.capacity() * mem::size_of::<RawVal>()
```

Comment:
In this case we need `heap_size` method for `RawVal` as well, since it one of its variants may contain `String` values allocated on the heap